karma.conf.js

```diff
@@ -1,30 +1,36 @@
-var webpackConfig = require('./webpack.config.js');
+var webpackConfig = require("./webpack.config.js");
 
 module.exports = function (karma) {
     karma.set({
-        plugins: ['karma-webpack', 'karma-chai', 'karma-sinon', 'karma-mocha', 'karma-chrome-launcher'],
+        plugins: [
+            "karma-webpack",
+            "karma-chai",
+            "karma-sinon",
+            "karma-mocha",
+            "karma-chrome-launcher",
+        ],
 
-        frameworks: ['chai', 'sinon', 'mocha'],
+        frameworks: ["chai", "sinon", "mocha", "webpack"],
 
         files: [
-            'src/**/*.js',
-            'test/**/*.js',
+            { pattern: "src/**/*.js", watched: false },
```

Comment:
hmm since we have so many Style changes here I guess it would make sense to add Prettier To the project so we can avoid running into some code style issues, I will prepare one Pr for that =)

Reply:
Go ahead! That's definitely a must 😋